components/meter/meter-radial.js

```diff
@@ -44,8 +44,12 @@ class MeterRadial extends LitElement {
 		const lengthOfLine = 115; // found by approximating half the perimeter of the ellipse with radii 38 and 35
 		const progressFill = this.value / this.max * lengthOfLine;
 		const secondaryTextElement = this.text ? html`<div class="d2l-body-small d2l-meter-radial-text">${this.text}</div>` : html``;
+		const label = this.text ? `${this.text}, progress indicator` : 'progress indicator';
```

Comment:
We will want to always have primary text to show too. `${this.value} / ${this.max}` in the middle - `${this.text}, ${this.value} / ${this.max}, progress indicator`